### DISCUSSION TITLE
Can you emulate touch input on Windows without a touchscreen?

### DISCUSSION THREAD

**Discussion by msneijders** [AvaloniaUI/Avalonia#14423] [Q&A] [answered]
While developing on Windows for touch devices (Android device, or touchscreen on desktop) without having a touchscreen on my dev machine, I would like to emulate touch input; so mouse clicks/moves will be handled like touch events, and I can test things like `PullGestureRecognizer`.

Is this currently possible with avalonia? it would shorten my dev cycle.

**Comment by maxkatz6** [comment, marked as answer]
No, but there was an idea to add a flag (possibly an attached property) that would configure behavior of default gesture recognizers to accept mouse as a touch input.
